src/stories/vocabulary_icons.stories.mdx

```diff
@@ -181,6 +138,44 @@ Using the `.colored` class, you can use a colored version of these icons.
   <IconItem name="cc-pd-alt">
     <i class="icon cc-pd-alt"></i>
   </IconItem>
+  <IconItem name="cc-letterheart">
+    <i class="icon cc-letterheart"></i>
+  </IconItem>
+</IconGallery>
+
+To have a white fill inside the icons, use the class `.has-background-white`. It's subtle but it's there.
```

Comment:
I tried this before opening the issue ;) It doesn't work. The white background has to be in the SVG inside the circle

<img width="733" alt="Screen Shot 2020-04-02 at 15 24 18" src="https://user-images.githubusercontent.com/707019/78284607-0ef5fb80-74f6-11ea-91d2-d0bd0a442c90.png">

Reply:
Nevermind. I see you added a background style

Reply:
But I did write `background-color: transparent !important;`. Why did the white color bleed out of the icons?